2-mandatory/2-water-bottle.js

```diff
@@ -19,18 +19,29 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    bottle.volume = 100;
```

Comment:
Your function needs to return something like this `return this.volume = 100` You need to use this key word to access the volume in your fillUp methods